vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -173,6 +168,10 @@ def save_guest_devices_inventory(hardware, hashes)
         # Save the hardware to force an id if not found
         hardware.save! if hardware.id.nil?
         h[:network][:hardware_id] = hardware.id
+      else
+        # ensure save_network_inventory gets called
+        # it will be deleted
+        h[:network] = {}
```

Comment:
@Fryguy sounds good.

[save_network_inventory](https://github.com/ManageIQ/manageiq/blob/master/vmdb/app/models/ems_refresh/save_inventory.rb#L197-L204) says that if the network hash key is not set, then clear out the stale `network` value. Yes, this is unlike all other `save_*_inventory` methods.

Before, the `save_*_inventory` methods were always called, whether the hashes was nil or not. After #1363, the methods are only called for non nil hashes.
So we are currently no longer calling into `save_network_inventory` with a nil hash and no longer clearing out stale network references.

This PR starts setting blanks (instead of nil) into the `hashes[:network]`, so `save_network_inventory` is called, and can clear out stale `network` reference once more.

call when you want to talk